### DISCUSSION TITLE
建议将人体识别之类的AI功能移动到单独的项目里或者删除

### DISCUSSION THREAD

**Discussion by qicfan** [Yuukiy/JavSP#270] [想法 | Ideas]
tensorflow包有1.3G，就为了裁切个图片，太不划算了。
可以考虑用公有云的接口（阿里云有类似的，不过不知道会不会进局子）
或者单独成一个项目打包比如AI，然后在主项目里的配置文件中添加AI的可执行文件路径来使用。
或者移除该功能吧。

![微信图片_20240405175528](https://github.com/Yuukiy/JavSP/assets/758655/50999f02-0adf-4969-b94a-ae707760fc5a)

**Comment by Yuukiy** [comment]
我也有这个想法，这个通常不会更新，每次都打包进去会导致release太大了，就因为这个最近一直没有打新的包

**Comment by glyh** [comment]
有一个解决方案就是用opencv重写，这个yunet看起来还挺小的。
https://github.com/opencv/opencv_zoo/tree/main/models/face_detection_yunet
之前作者为什么把opencv的模块删除了？

#284
